Astro 3/21 release notes (#2033)
astronomer/docs: astro/release-notes.md

**Latest CLI Version**: 1.11.0 ([Release notes](cli/release-notes.md))

## March 21, 2023

### Automate Workspace and Deployment actions using Workspace API tokens

> Workspace API tokens can be used to automate actions such as adding users to a Workspace and creating new Deployments, as well as everything you can automate with a [Deployment API key](api-keys.md). You can customize the role and expiration date of the token to give it the minimum required permissions for the task it's completing.

Reviewer: ```suggestion
Use Workspace API tokens to automate actions, such as adding users to a Workspace and creating new Deployments, or for processes that a [Deployment API key](api-keys.md) can automate. You can customize the role and expiration date of the token to give it the minimum required permissions for the task it completes.
```
First sentence felt a little awkward. Thoughts?

> To create and use Workspace API tokens, see [Workspace API tokens]  (link to follow).

Author: ```suggestion
To create and use Workspace API tokens, see [Workspace API tokens](workspace-api-tokens.md).
```
